lib/ansible/executor/module_common.py

```diff
@@ -1188,7 +1188,9 @@ def _find_module_utils(module_name, b_module_data, module_path, module_args, tas
                     if not os.path.exists(lookup_path):
```

Comment:
Right, but we can't merge or backport that right now.

Reply:
Assuming py3 is the only thing being supported, why wouldn't we be able to backport that? Given https://github.com/python/cpython/blob/main/Lib/os.py#L213, the method is running the same check?

Reply:
more for my curiosity than anything else?

Reply:
The Python >=3.8 requirement is only for Ansible >= 2.12 (current `devel`). Previous version still support Python 2.

Reply:
We can't merge this change currently because in our CI, the Ansible control node and the remote node are the same. For systems that only have Python 2 (such as CentOS 6) or older versions of Python 3, tests will fail (which is what is happening currently).

Reply:
@samdoran for now you could just point it at makedirs_safe() and when controller spilt happens, i'll update in  #75108 with the rest of makedirs_safe usage

Reply:
I added a not that we can switch to this method once split controller/remote is merged.

Reply:
@cloudnull for more detail: the `ansible-test` changes that support split testing on the controller and remotes is landing in devel Real Soon Now, but until it does, we can't merge anything that doesn't work back to Python 2.6 without breaking all testing on those targets or disabling all module tests on < py3.8. It's a BIG change to the internals of the way all that stuff works, and breaks a lot of assumptions that module tests often make about the test target host being the same as the controller host. Since lots of external collections rely on `ansible-test`, we're taking time to maximize backwards compatibility to keep their tests working, rather than just slamming in the changes we need and letting the community pick up the pieces (otherwise it would've landed long ago in 2.11).